app/views/errorPages/templates/ErrorTemplate.scala.html

```diff
@@ -14,11 +14,11 @@
  * limitations under the License.
  *@
 
-@import views.html.layouts.layout
+@import views.html.layouts.unifiedLayout
 @import views.html.components._
 
 @this(
-      mainTemplate: layout,
+      mainTemplate: unifiedLayout,
```

Comment:
We need to pass isAgent to this template as well as we are using it in both Individual and Agent.

Reply:
Thanks it's been done